bin/postgres-ha/kube-ssh-wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+source "/tmp/pod_env.sh"
+
+CLUSTER_LABEL="pg-cluster=${CLUSTER_NAME}"
+TARGET_LABEL="pgo-backrest-repo=true"
+
+opts=$(echo "$@" | grep -o "\-\-c.*")
+pod=$(kubectl get pods --selector=${CLUSTER_LABEL},${TARGET_LABEL} -o name)
+
+exec kubectl exec -i "${pod}" -- bash -c "pgbackrest ${opts}"
```

Comment:
@andrewlecuyer Note that we may need the `oc` equivalent here -- not for this commit.